src/Abstractions/Internal/IOrchestrationSubmitter.cs

```diff
@@ -40,5 +41,6 @@ public interface IOrchestrationSubmitter
         TaskName orchestratorName,
         object? input = null,
         StartOrchestrationOptions? options = null,
+        HashSet<string>? orchestrationIdReusePolicy = null,
```

Comment:
"ReusePolicy" isn't that clean to me. Was the name `InstanceIdConflictBehavior` considered?